.github/workflows/windows.yaml

```diff
@@ -0,0 +1,19 @@
+name: "windows-testing"
+
+on:
+  push:
+    branches:
+      - "windows-testing"
```

Comment:
When you're ready to merge, would you make sure the `on` triggers match those in the existing ci.yaml?

Reply:
If that's the goal, should this be another job in ci.yaml rather than a separate workflow?

Reply:
👍 good idea!

Reply:
ok

Reply:
I won't merge this pr until #432 goes in.